app/thirdparty/res/drawable/ic_baseline_arrow_downward_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
add all android ic to main, not thirdparty

Reply:
Moved

Reply:
![grafik](https://user-images.githubusercontent.com/6735650/96711560-0b489a00-139e-11eb-9042-df2c5f2d0db4.png)

@harshad1 
Are you sure you have commited that? Still shows up thirdparty here

Reply:
I think this may have been a problem with github's display. Should show correctly now